blame.go

```diff
@@ -13,17 +13,21 @@ import (
 	"srcd.works/go-git.v4/utils/diff"
 )
 
+// BlameResult represents the result of a Blame operation
```

Comment:
Add a full stop at the end of the sentence, as it is going to be rendered as English prose by go-doc.